### ISSUE TITLE
Failure finding main SourceSet

### ISSUE THREAD

**Issue by es0329** [patxibocos/poetimizely#48]
Version: `1.0.0-beta02`

The `./gradlew poetimize` task fails with the error message:
```
Execution failed for task ':app:poetimize'.
> SourceSet with name 'main' not found.
```

The `./gradlew sourceSets` task indicates a main exists:
```
main
----
Compile configuration: compile
build.gradle name: android.sourceSets.main
Java sources: [app/src/main/java, app/src/main/kotlin]
Manifest file: app/src/main/AndroidManifest.xml
Android resources: [app/src/main/res]
Assets: [app/src/main/assets]
AIDL sources: [app/src/main/aidl]
RenderScript sources: [app/src/main/rs]
JNI sources: [app/src/main/jni]
JNI libraries: [app/src/main/jniLibs]
Java-style resources: [app/src/main/resources]
```

Let me know if there's other info I may provide to help.

**Comment by patxibocos** [comment]
Hi Eric, let me try to reproduce it and figure out the fix. Thanks a lot for the report 🙂

**Comment by patxibocos** [comment]
Hello Eric, note that I have opened the PR #49 which fixes this issue.
The problem happens because getting the main source set on an Android project fails so the easy fix I am introducing is getting `src/main/kotlin` or `src/main/java` folders as fallbacks.

**Comment by patxibocos** [comment]
The issue should be solved as part of the `1.0.0-beta03` version.
You should be able to verify it using the latest [Gradle plugin](https://plugins.gradle.org/plugin/com.patxi.poetimizely) version.

**Comment by es0329** [comment]
Thank you @patxibocos. I can confirm your changes fixed that issue.
